metadetect/fitting.py

```diff
@@ -213,6 +216,12 @@ def _fit_obs(
         res["obj_res"] = fitter.go(obs)
         res["psf_res"] = fitter.go(obs.psf, **psf_go_kwargs)
 
+        if fitter.kind == 'am':
+            res["obj_res"]["mom"] = res["obj_res"]["sums"]
+            res["obj_res"]["mom_cov"] = res["obj_res"]["sums_cov"]
+            res["psf_res"]["mom"] = res["psf_res"]["sums"]
+            res["psf_res"]["mom_cov"] = res["psf_res"]["sums_cov"]
+
```

Comment:
Are the normalizations here the same? We may want to upstream this change into ngmix itself.

Reply:
the AM sums are not normalized by flux

Reply:
Is that the same as prepsfmom?

Reply:
Yes that should be the same. I will double check.

Reply:
These norms are the same. We should move this bit of code into ngmix.